tests/unit/testsuite/Bolt/Boltpay/controllers/ApiControllerTest.php

```diff
@@ -121,11 +122,14 @@ public function testHookAction_thatOrderNotProcessedForIrreversiblyRejectedHooks
             || $payment->getAuthorizationTransaction()
             || $payment->getLastTransId()
         );
-
-        $apiControllerMock
-            ->expects($this->once())
-            ->method('sendResponse')
-            ->with($this->equalTo(200));
+        $this->markTestIncomplete(
+            'method handleFailedPaymentHook has exit'
+            );
+        
+//         $apiControllerMock
+//             ->expects($this->once())
+//             ->method('sendResponse')
+//             ->with($this->equalTo(200));
```

Comment:
Please delete unneeded code instead of commenting it out. If there's a reason why this needs to be commented out and kept in the code, please write a comment.

Reply:
@adentong I could not delete this, because this test is marked as Incomplete and this line broke test - it must be fixed in future, not hidden

Reply:
let's add comment like
`// TODO: fix test`
and remove code for now. in next PR you can add back this code

Reply:
Ok. Then can you write a comment about it? I don't want anyone to see commented out code and think it's unneeded and delete it by accident.

Reply:
What Kazuki suggests also works.

Reply:
Ok, i removed all comments - in all cases it need to rewrite that tests.... and methods too